index.html

```diff
@@ -0,0 +1,167 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tesfaye-capstone-project</title>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&family=Poppins&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css" />
+
+</head>
+<body>
+    <div class="header-container">
```

Comment:
- [ OPTIONAL] Kindly make sure your popup fills the whole screen to avoid any bugs :+1: 
![Screenshot from 2023-02-17 23-44-06](https://user-images.githubusercontent.com/97834160/219800029-e5c32bdb-ec05-4a5a-a12e-a69903f753be.png)